db/ticker.go

```diff
@@ -0,0 +1,45 @@
+package db
+
+import (
+	"github.com/trustwallet/watchmarket/db/models"
+	"github.com/trustwallet/watchmarket/market"
+	"github.com/trustwallet/watchmarket/pkg/watchmarket"
+	"strconv"
+)
+
+func (i *Instance) AddTickers(tickers []watchmarket.Ticker, provider market.Provider) error {
+	var errorsList []error
+
+	for _, ticker := range tickers {
```

Comment:
add batch?

Reply:
I want to try it without batch, if there will be problem with it - will change to the batch, it is not a problem. Just want to have a bit better readability

Reply:
👍